release-notes/opensearch-release-notes-2.14.0.md

```diff
@@ -0,0 +1,825 @@
+* Fix create acceleration bugs ([#1599](https://github.com/opensearch-project/dashboards-observability/pull/1599))
+* Updating usePolling to cleanup after unmount ([#1598](https://github.com/opensearch-project/dashboards-observability/pull/1598))
+* Enable integration install flyout for other install buttons ([#1596](https://github.com/opensearch-project/dashboards-observability/pull/1596))
+* Disable close button when integration is being installed ([#1591](https://github.com/opensearch-project/dashboards-observability/pull/1591))
+
+
+### Opensearch Alerting
+
+
+* Fix fieldLimit exception in docLevelMonitor. ([#1368](https://github.com/opensearch-project/alerting/pull/1368))
+
+
+### Opensearch Alerting Dashboards Plugin
+
+
+* Include server.basepath config in the manage notifications channel Url. ([#914](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/914))
+* Removed cross cluster monitor experimental banner, and fixed bugs. ([#933](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/933))
+* Fetching timezone from ui settings for Trigger context formatting ([#913](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/913))
+
+
+### Opensearch Anomaly Detection Dashboards
+
+
+* Populate selected indices from query params on initial load ([#713](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/713))
+* Build query parameters using data\_end\_time ([#741](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/741))
+
+
+### Opensearch Asynchronous Search
+
+
+* Fix number of total shards in partial search response ([#565](https://github.com/opensearch-project/asynchronous-search/pull/565))
+
+
+### Opensearch Cross Cluster Replication
+
+
+* Handle response for deletion of non-existent autofollow replication rule ([#1371](https://github.com/opensearch-project/cross-cluster-replication/pull/1371))
+
+
+### Opensearch Dashboards Maps
+
+
+* Fix zoom level type error in custom layer ([#605](https://github.com/opensearch-project/dashboards-maps/pull/605))
+
+
+### Opensearch Dashboards Notifications
+
+
+* Handle error state when dataSource switches to invalid dataSource ([#199](https://github.com/opensearch-project/dashboards-notifications/pull/199))
+* Fix broken osd functional test repo ([#189](https://github.com/opensearch-project/dashboards-notifications/pull/189))([#190](https://github.com/opensearch-project/dashboards-notifications/pull/190))
+
+
+### Opensearch Dashboards Search Relevance
+
+
+* Added flag for fix when mds is disabled ([#390](https://github.com/opensearch-project/dashboards-search-relevance/pull/390)) ([#392](https://github.com/opensearch-project/dashboards-search-relevance/pull/392))
+
+
+
+### Opensearch Flow Framework
+
+
+* Reset workflow state to initial state after successful deprovision ([#635](https://github.com/opensearch-project/flow-framework/pull/635))
+
+
```

Comment:
Extra whitespace here that seems to make the spacing different than for everything else

Reply:
Ack!